### PR TITLE
fix: formatting in `numbat-wasm/www/`

### LINKED ISSUE #456: `numbat-wasm/www/` has formatting issues
It'd be nice if we could clean up the formatting in `numbat-wasm/www/`, and just focus on functional changes afterward.

### PR DESCRIPTION
**Pull request by hamirmahal**
fixes #456.

### CODE REVIEW

File: numbat-wasm/www/terminal.css
Comment (hamirmahal) [inline numbat-wasm/www/terminal.css:1]: We should auto-format any file we plan on editing. I'm unsure if `numbat-wasm/www/terminal.css` fits that criteria.

**Review by sharkdp — CHANGES_REQUESTED** [review]
I'm okay with auto-formatting `main.css`, `index.js` and `index.html`. Please revert all of the other changes, including the new files. I'd like to keep this minimal.

**Comment by hamirmahal** [comment]
What're your thoughts on using https://github.com/biomejs/biome to lint and format web code?

It's written in Rust.

**Comment by sharkdp** [comment]
> What're your thoughts on using https://github.com/biomejs/biome to lint and format web code?

But that's only for JavaScript, right?

**Comment by hamirmahal** [comment]
> > What're your thoughts on using https://github.com/biomejs/biome to lint and format web code?
> 
> But that's only for JavaScript, right?

I'm not too sure. I haven't used it before, but I can look into it if you're interested in integrating it with `numbat-wasm/www/`.

**Comment by sharkdp** [comment]
Closed via 8ce1eea5c38f7f52661cdf7603290232135f340e

**Pull request closed without merging**